.github/workflows/e2e-test.yaml

```diff
@@ -0,0 +1,38 @@
+name: End to End test
+
+on:
+  push:
+  # Run on branch/tag creation
+  create:
+  # Run on pull requests
+  pull_request:
+
+jobs:
+  create-cluster:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Astarte Cluster
+        id: astarte
+        uses: astarte-platform/astarte-cluster-action@v1
+      - name: Checkout sources
+        uses: actions/checkout@v2
+      - name: Install interface
+        run: | 
+          astartectl realm-management interfaces install $GITHUB_WORKSPACE/examples/interfaces/com.test.Everything.json
```

Comment:
For tests, it is better to use the "canonical" prefix for interface names: `org.astarte-platform`...